dbt/include/athena/macros/materializations/snapshots/snapshot.sql

```diff
@@ -15,7 +15,7 @@
     If hive table then Recreate the snapshot table from the new_snapshot_table
     If iceberg table then Update the standard snapshot merge to include the DBT_INTERNAL_SOURCE prefix in the src_cols_csv
 #}
-{% macro hive_snapshot_merge_sql(target, source, insert_cols, table_type) -%}
+{% macro hive_snapshot_merge_sql(target, source, insert_cols) -%}
```

Comment:
as you change macro definition, do we need to fix it where macro is actually called ?

Reply:
I searched for all invocations of this macros. And this is the one which I found:
`{% set final_sql = hive_snapshot_merge_sql(
                target = target_relation,
                source = new_snapshot_table
             )
          %}`
I deleted insert_cols param also.

Reply:
thanks for pointing this out. however we prefer to keep PRs as atomic as it's possible. Would you mind to move this change to another PR because it does not belong to the feature. sorry for annoying comments 😄

Reply:
Sure. It is not a problem. Fixed.